app/models/user.rb

```diff
@@ -14,6 +14,8 @@ class User < ApplicationRecord
   has_many :journals
   has_many :assignments, inverse_of: 'creator'
   has_many :feedbacks, inverse_of: 'author'
+  has_many :trial_feedbacks, inverse_of: 'author'
+  has_many :trial_feedbacks, inverse_of: 'reviewer'
```

Comment:
You can't define the same association twice, the second definition will replace the first one.